worlds/kh2/Client.py

```diff
@@ -844,10 +842,57 @@ async def verifyItems(self):
             logger.info(e)
             logger.info("line 840")
 
+    def get_addresses(self):
+        if not self.kh2connected and self.kh2 is not None:
+            if self.kh2_game_version is None:
+
+                if self.kh2_read_string(0x09A9830, 4) == "KH2J":
+                    self.kh2_game_version = "STEAM"
+                    self.Now = 0x0717008
+                    self.Save = 0x09A9830
+                    self.Slot1 = 0x2A23518
+                    self.Journal = 0x7434E0
+                    self.Shop = 0x7435D0
+                elif self.kh2_read_string(0x09A92F0, 4) == "KH2J":
+                    self.kh2_game_version = "EGS"
+                else:
+                    if self.game_communication_path:
+                        logger.info("Game Version is not 0.9. Checking with most up to date addresses of github. If file is not found will be downloading datafiles. This might take a moment")
+                        #if mem addresses file is found then check version and if old get new one
+                        if not os.path.exists(self.game_communication_path + f"\kh2memaddresses.json"):
+                            mem_resp = requests.get("https://raw.githubusercontent.com/JaredWeakStrike/KH2APMemoryValues/master/kh2memaddresses.json")
+                            if mem_resp.status_code == 200:
+                                self.mem_json = json.loads(mem_resp.content)
+                                with open(os.path.join(self.game_communication_path, f"kh2memaddresses.json"),
+                                        'w') as f:
+                                    f.write(json.dumps(self.mem_json, indent=4))
+                        else:
+                            with open(self.game_communication_path + f"\kh2memaddresses.json", 'r') as f:
+                                self.mem_json = json.load(f)
+                        for key, value in self.mem_json.items():
+
+                            if self.kh2_read_string(eval(self.mem_json[key][0]["GameVersionCheck"]), 4) == "KH2J":
+                                self.Now = eval(self.mem_json[key][0]["Now"])
+                                self.Save=eval(self.mem_json[key][0]["Save"])
+                                self.Now = eval(self.mem_json[key][0]["Slot1"])
+                                self.Now = eval(self.mem_json[key][0]["Journal"])
+                                self.Now = eval(self.mem_json[key][0]["Shop"])
```

Comment:
Are these all supposed to be `self.Now` ?

Reply:
no and I have no idea how I didnt change it. I think I had them as being correct after testing then changed them to test somthing else and forgot to change them back. thanks for bringing this to my attention and I just pushed a change that fixed it